### PR TITLE
[feature] piv: manage message at stop

### PR DESCRIPTION
**Pull request by patochectp**
ref: ND-1097

### CODE REVIEW

File: kirin/piv/model_maker.py [hunk 1]
```diff
@@ -197,6 +197,25 @@ def _check_stop_time_consistency(previous_rt_stop_time_dep, current_rt_stop_time
...
+    departure_stop = get_value(arret, "depart", nullable=True)
+    motif = None
+    if departure_stop:
+        motif = get_value(departure_stop, "motifModification", nullable=True)
+    if not motif and arrival_stop:
+        motif = get_value(arrival_stop, "motifModification", nullable=True)
+    if not motif and departure_stop:
+        event = get_value(departure_stop, "evenement", nullable=True)
+        if event:
+            motif = get_value(event, "texte", nullable=True)
```
Comment (pbougue) [inline kirin/piv/model_maker.py:211]: minor: Like you did for other nullable fields, you can use get:
```suggestion
        event = departure_stop.get("evenement", {}).get("texte", None)
```

File: tests/fixtures/piv/stomp_20201022_23186_delayed_5min.json [hunk 2]
```diff
@@ -1,973 +1,979 @@
...
+                        "code":"85010157"
+                     },
+                     "rang":1,
+                     "dureeStationnement":0,
+                     "dureeStationnementReelle":0
+                  },
+                  {
+                     "arrivee":{
+                        "evenement":{
+                           "texte":"Absence inopinée d'un agent () evenement arrivee",
```
Comment (pbougue) [inline tests/fixtures/piv/stomp_20201022_23186_delayed_5min.json:112]: :eagle: 
```suggestion
                           "texte":"Absence inopinée d'un agent (evenement arrivee)",
```

File: tests/fixtures/piv/stomp_20201022_23186_delayed_5min.json [hunk 3]
```diff
@@ -1,973 +1,979 @@
...
+                        "dateHeureReelle":"2020-10-22T22:52:00+02:00",
+                        "sourceHoraire":"CO",
+                        "typeAffichage":"HORAIRE",
+                        "indicateurAdaptation":false,
+                        "planTransportSource":"PTA",
+                        "motifModification":""
+                     },
+                     "depart":{
+                        "evenement":{
+                           "texte":"Absence inopinée d'un agent evenement depart",
```
Comment (pbougue) [inline tests/fixtures/piv/stomp_20201022_23186_delayed_5min.json:337]:
```suggestion
                           "texte":"Absence inopinée d'un agent (evenement depart)",
```

**Review by pbougue — APPROVED** [review]
I think it's better to have "clean" fixtures as we are about to copy/paste/modify them.
:heavy_check_mark: But it's fine anyway.

**Review by pbougue — APPROVED** [review]

File: kirin/piv/model_maker.py (hunk 1 above)
Comment (patochectp) [reply kirin/piv/model_maker.py]: done in 666e77bb3b7382474597d6a1f577eab3a2ee0eae

**Review by woshilapin — APPROVED** [review]

**Pull request merged**